components/Card/Thumbnail.d.ts

```diff
@@ -1,12 +1,14 @@
-import { ComponentType } from 'react';
+import { FC } from 'react';
 
-type Thumbnail = ComponentType<{
+export interface ThumbnailProps {
     src: string;
     alt: string;
     rounded?: boolean;
     theme?: { colors?: object };
-}> & {
-    displayName: 'Card.Thumbnail';
+}
+
+declare const Thumbnail: FC<ThumbnailProps> & {
+    displayName: "Card.Thumbnail";
```

Comment:
Is it really necessary to declare displayName here?

Reply:
I mistakenly thought that maybe it would help with debugging typescript applications, but I see that it doesn't. I will remove